vite.config.ts

```diff
@@ -2,21 +2,25 @@ import react from '@vitejs/plugin-react'
 import { configDotenv } from 'dotenv'
 import path from 'node:path'
 import { defineConfig } from 'vite'
-import { viteExternalsPlugin } from 'vite-plugin-externals'
+import { cdn } from 'vite-plugin-cdn2'
+import { unpkg } from 'vite-plugin-cdn2/url.js'
 
 configDotenv()
 
 // https://vitejs.dev/config/
-export default defineConfig(async env => ({
+export default defineConfig(async _=> ({
   base: `/${process.env.BASE_URL || 'app'}/`,
   plugins: [
     react(),
-    env.mode === 'production' ? viteExternalsPlugin({
-      '@babel/standalone': 'Babel',
-      'react': 'React',
-      'react-dom': 'ReactDOM',
-      'sentinel-js': 'sentinel'
-    }) : null
+    cdn({
```

Comment:
The reason why I set the production condition here is because I want to have as detailed a stack as possible in the development environment. Is this wrong? 🤔️

Reply:
No. because `cdn` plugin only built on build stage. ( unless you set `apply:server` for it.

Reply:
so cool! do it!